packages/amqp/README.md

```diff
@@ -0,0 +1,63 @@
+# AMQP (Advanced Message Queuing Protocol)
+
```

Comment:
can you also add link to https://github.com/kibertoad/message-queue-toolkit/blob/main/README.md for transport-agnostic library documentation?